bugwarrior/services/linear.py

```diff
@@ -17,19 +17,22 @@ class LinearConfig(config.ServiceConfig):
     api_token: str
 
     host: config.StrippedTrailingSlashUrl = config.StrippedTrailingSlashUrl(
-        "https://api.linear.app/graphql", scheme="https", host="api.linear.app"
+        "https://api.linear.app/graphql"
     )
-    statuses: config.ConfigList = config.ConfigList([])
+    statuses: config.ConfigList = []
     status_types: typing.Optional[config.ConfigList] = None
     import_labels_as_tags: bool = False
     label_template: str = "{{label|replace(' ', '_')}}"
     also_unassigned: config.UnsupportedOption[bool] = False
 
-    @pydantic.v1.root_validator
+    @model_validator(mode='before')
+    @classmethod
     def statuses_or_status_types(cls, values):
-        if values["statuses"] and values["status_types"]:
+        statuses = values.get("statuses", [])
```

Comment:
Doesn't seem like there's any reason for this one to get a different default than the next one. They're both only used in boolean expressions.

```suggestion
        statuses = values.get("statuses")
```